### DISCUSSION TITLE
Is there a way to change the filename of changelog file?

### DISCUSSION THREAD

**Discussion by tukusejssirs** [jscutlery/semver#786] [Q&A] [answered]
I prefer naming my files in lowercase (or camelCase or kebab-case) instead of all caps.

Is there an option to rename changelog file to `changelog.md`?

Note that I use synced mode.

**Update:** I believe there is no such option, as `CHANGELOG.md` is hard-coded in `packages/semver/src/generators/install/utils/create-changelog.ts` (L4):

https://github.com/jscutlery/semver/blob/539e6cdaecaf6564911ab1aa83c2a77ea1d52147/packages/semver/src/generators/install/utils/create-changelog.ts#L4

**Comment by tukusejssirs** [comment]
I added this feature in my fork (in [`changelog-filename`](https://github.com/tukusejssirs/semver/tree/changelog-filename) branch). No tests were created, as [`packages/semver/src/executors/version/utils/write-changelog.spec.ts`](https://github.com/tukusejssirs/semver/blob/015f9415d410043722d363f013ae5f22c726a27f/packages/semver/src/executors/version/utils/write-changelog.spec.ts#L11) is commented out, therefore, I am not sure how you want me to proceed. Nevertheless, my changes pass all uncommented tests (`yarn test` or actually `yarn jest`).

Also, I wanted to lint the code using `yarn lint`, however, it failed to run as Nx `[could not] find configuration for task semver:workspace-lint`. And when I have run `yarn nx lint`, it has failed to throw an error in `node_modules/@commitlint/cli/lib/cli.js` (L123), and it output a warning that the `command "yarn commitlint --from=last-release" exited with non-zero status code`. I did not investigate further.

I did not forget to run `yarn format`.

I’d create a PR for this if you are interested, however, I’d like know how do you want to deal with the tests.

**Reply by tukusejssirs** [reply]
@edbzn, sorry for pinging you, however, when you have some time, could you tell me what you think about this please? Thanks! :pray:

**Reply by edbzn** [reply]
You can run tasks using the Nx CLI: yarn nx lint semver, yarn nx test semver, yarn nx e2e semver.

**Comment by edbzn** [comment, marked as answer]
Hi, yeah the filename is hardcoded but we are in the situation where we don't want to bring new options to semver as we already have many. Let's wait and see if other users might need it.
